docs/user/ppl/functions/datetime.rst

```diff
@@ -1034,6 +1064,35 @@ Example::
     +------------------------------------------------+----------------------------------+------------------------------------------------+
 
 
+SUBTIME
+-------
+
+Description
+>>>>>>>>>>>
+
+Usage: subtime(expr1, expr2) subtracts expr2 from expr1 and returns the result. If argument is TIME, today's date is used; if argument is DATE, time at midnight is used.
+
+Argument type: DATE/DATETIME/TIMESTAMP/TIME, DATE/DATETIME/TIMESTAMP/TIME
+
+Return type map:
+
+(DATE/DATETIME/TIMESTAMP, DATE/DATETIME/TIMESTAMP/TIME) -> DATETIME
+
+(TIME, DATE/DATETIME/TIMESTAMP/TIME) -> TIME
+
+Antonyms: `ADDTIME`_
+
+Example::
+
+    os> source=people | eval `'2008-12-12' - 0` = SUBTIME(DATE('2008-12-12'), DATE('2008-11-15')), `'23:59:59' - 0` = SUBTIME(TIME('23:59:59'), DATE('2004-01-01')), `'2004-01-01' - '23:59:59'` = SUBTIME(DATE('2004-01-01'), TIME('23:59:59')), `'10:20:30' - '00:05:42'` = SUBTIME(TIME('10:20:30'), TIME('00:05:42')) | fields `'2008-12-12' - 0`, `'23:59:59' - 0`, `'2004-01-01' - '23:59:59'`, `'10:20:30' - '00:05:42'`
+    fetched rows / total rows = 1/1
+    +---------------------+------------------+-----------------------------+---------------------------+
+    | '2008-12-12' - 0    | '23:59:59' - 0   | '2004-01-01' - '23:59:59'   | '10:20:30' - '00:05:42'   |
+    |---------------------+------------------+-----------------------------+---------------------------|
+    | 2008-12-12 00:00:00 | 23:59:59         | 2003-12-31 00:00:01         | 10:14:48                  |
+    +---------------------+------------------+-----------------------------+---------------------------+
```

Comment:
I would add test that jumps over month/day or year date.

Reply:
Fixed in db18570.